src/Components/UnsupportedComponentRenderer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Components;
+
+use Exception;
+use Tempest\Console\Console;
+use Tempest\Console\ConsoleComponent;
+
+final class UnsupportedComponentRenderer implements ComponentRenderer
```

Comment:
We use this one while running tests and for commands running in the background. I do plan on adding a component renderer for testing, but that's out of scope for this PR